applications/luci-app-lldpd/root/usr/libexec/rpcd/lldpd

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env lua
```

Comment:
This rpcd script is still implemented in lua and requires `luci-lua-runtime` to run correctly. You may need to either:

* add `luci-lua-runtime` in dependency
* rewrite this via ucode since `luci-base` already depends on ucode

Reply:
Thanks for the hint! I have added the Lua dependencies in https://github.com/openwrt/luci/pull/6456/commits/6a756595edaa97598d3c2a68df1b2a6fd6f35389.